frontend/src/components/DetailsTable.tsx

```diff
@@ -82,7 +82,8 @@ export default (props: DetailsTableProps) => {
                   <span className={css.key}>{key}</span>
                   <Editor
                     width='100%'
-                    height='300px'
+                    minLines={3}
+                    maxLines={20}
```

Comment:
How is 20 determined?
Would this number need to be changed depending on user's screen size?

Reply:
Great Thanks @Bobgy for your review,  20 line defined here to avoid the jsonEditor Window is too big, If the lines is greater than 20, then scroll bar will be appearred for user to view all. So user don't need to set it again for different screen size.